### ISSUE TITLE
Cannot Parse settings...

### ISSUE THREAD

**Issue by Emilot** [jcorporation/myMPD#179]
**Describe the bug**
Mympd is not loading in a clean Archlinux image

**To Reproduce**
Run ./build.sh and after that i enable the mympd.service
Start service and it throws this message...

![mympd_2](https://user-images.githubusercontent.com/18086896/67306822-405aac00-f500-11e9-8b52-8fe5a9073870.jpg)

**Expected behavior**
To open the GUI

**Screenshots**
These are the errors...

![mympd_1](https://user-images.githubusercontent.com/18086896/67306892-58cac680-f500-11e9-9c50-1090af3114a4.jpg)

and during the building i have a strange error "clock skew detected"

![mympd_3](https://user-images.githubusercontent.com/18086896/67306939-697b3c80-f500-11e9-976e-212afb45a2d4.jpg)

**Desktop (please complete the following information):**
 - OS: ArchlinuxArm
 - Browser all

**Additional context**
It happens with latest Master and latest Devel...

![mympd_4](https://user-images.githubusercontent.com/18086896/67307129-acd5ab00-f500-11e9-9a81-ea5dd0b5efb9.jpg)

Thank you JC!

**Comment by jcorporation** [comment]
Please use master. Devel is at the moment unstable, because of the sds branch merge.

A clock skew can explain the backend and the last JavaScript error. The other javascript errors are not from myMPD.

Remove the source tree, check the clock and please try it again.

**Comment by Emilot** [comment]
JC i fix the time etc and the error is gone, but....

i have this:


Oct 22 22:37:31 archphile mympd[3188]: INFO     /root/myMPD/src/mpd_client.c:3748: Updated smart playlist myMPDsmart-bestRated with 0 songs, minValue: 0
Oct 22 22:37:31 archphile mympd[3188]: INFO     /root/myMPD/src/mpd_client.c:3748: Updated smart playlist myMPDsmart-mostPlayed with 0 songs, minValue: 0
Oct 22 22:37:31 archphile mympd[3188]: INFO     /root/myMPD/src/mpd_client.c:3778: Updated smart playlist myMPDsmart-newestSongs
Oct 22 22:37:32 archphile mympd[3188]: DEBUG    /root/myMPD/src/web_server.c:197: Setting music_directory to /var/lib/mpd/music
Oct 22 22:37:32 archphile mympd[3188]: DEBUG    /root/myMPD/src/web_server.c:203: Setting rewrite_patterns to /pics/=/var/lib/mympd/pics,/library/=/var/lib/mpd/music
Oct 22 21:36:59 archphile mympd[3188]: DEBUG    /root/myMPD/src/web_server.c:273: New connection id 2
Oct 22 21:36:59 archphile mympd[3188]: VERBOSE  /root/myMPD/src/web_server.c:278: New websocket request (2): /ws/
Oct 22 21:36:59 archphile mympd[3188]: VERBOSE  /root/myMPD/src/web_server.c:287: New Websocket connection established (2)
Oct 22 21:37:00 archphile systemd[1]: mympd.service: Main process exited, code=killed, status=4/ILL
Oct 22 21:37:00 archphile systemd[1]: mympd.service: Failed with result 'signal'.

and the screen stays at "Connect to websocket" and mympd service is has already stopped as it seems at the text above...

**Comment by jcorporation** [comment]
Strange, you tried current master? Can you test it with latest release tarball? Is this a fresh install or is there a old /var/lib/mympd?

**Comment by Emilot** [comment]
I tried with 5.6.1, and current Master and Devel

I think in 5.6.0 it should play just fine.

Clean install on a newly Archlinux image environment.

**Comment by Emilot** [comment]
With 5.6.0 (from releases) i have this errors...

![mympd_7](https://user-images.githubusercontent.com/18086896/67326114-c1c13700-f51e-11e9-9942-2e8cb19fe286.jpg)

**Comment by jcorporation** [comment]
This is a known bug in 5.6.0, if no java is found at build time. Try to remove /etc/mympd.conf before installing 5.6.2.

**Comment by archphile** [comment]
Hi JC,
I got a little bit confused with your reference to java. I see java and perl as build dependencies but they do not exist in archlinux pkgbuild.

I also have the same problem with Emilot. Should we revise the PKGBUILD with:

makedepends=('cmake' 'jre-openjdk' 'perl')

?

Greetings,
M.

**Comment by jcorporation** [comment]
Perl and java are only needed for the build.sh script. The package builds should work without it.

**Comment by Emilot** [comment]
JC with 
    compile myMPD with ./build.sh memcheck
    run myMPD with catchsegv debug/mympd

it runs flawlessly... Everything seems fine...

but even if i install java and perl, i get the same error after building.

Really weird :(

**Comment by jcorporation** [comment]
Do you cleaned up the browser and the application cache?

**Comment by Emilot** [comment]
JC it was on a different computer. First time with MyMPD, so no cache...:(

Versions 5.6.2, 5.6.1,5.6.0...

Only the 5.5.3 plays just fine out of the box...

**Comment by jcorporation** [comment]
Error is the same as in the last screenshot?

**Comment by Emilot** [comment]
This one...

https://user-images.githubusercontent.com/18086896/67319160-3becbe00-f515-11e9-84ba-f01655f968b8.jpg

**Comment by jcorporation** [comment]
SSL is disabled? Please at attach the output of the mympd process and the network view from the webconsole.

**Comment by jcorporation** [comment]
The message "Error minifying" results from no java installation is found. The fallback copy action should be bug free since 5.6.1. 

/etc/mympd folder plays no role, myMPD sind 5.6.0 uses /etc/mympd.conf.

I am not sure what was the root cause of your initial problems, but I think it was a mix of the clock skew, caching issues and the bug in 5.6.0.

**Comment by Emilot** [comment]
I do not think so JC. Except the first rookie error with the time, even one person more has the same issue and went from 5.5.3 to 5.6.2...

I must try one more time in a clean Archlinux environment to see where is the problem and inform you.

**Comment by jcorporation** [comment]
Thats would be greate!

**Comment by sinkintothevoid** [comment]
I just tried to build using the PKGBUILD. 

I completely removed previous version and made sure that I had no leftovers. Browser cache was also cleared.

I Installed the latest package, I edited mympd.conf, I disabled ssl and manually set host to 127.0.0.1.

Whatever I do I keep getting the "Websocket connection failed error" and then mympd stops.

There is no log file under /var/log to share with you. 

What I also tried was to run mympd through console (without the service) and this is what I get:

https://i.imgur.com/vvkrmHn.png
